setup.cfg

```diff
@@ -58,12 +58,12 @@ dev =
     ipdb==0.13.13
```

Comment:
Put this in a separate commit with the title: ``⬆️(dependencies) update python dependencies``

Reply:
My bad ^^. I squashed my commits with the boot one.

Done